docs/extensions/get-started/anatomy.md

```diff
@@ -38,13 +38,15 @@ It contains a mix of Node.js fields, including scripts and dependencies, and Len
 Some of the most-important fields include:
 
 - `name` and `publisher`: Lens uses `@<publisher>/<name>` as a unique ID for the extension.
-For example, the Hello World sample has the ID `@lensapp-samples/helloworld-sample`.
-Lens uses this ID to uniquely identify your extension.
+  For example, the Hello World sample has the ID `@lensapp-samples/helloworld-sample`.
+  Lens uses this ID to uniquely identify your extension.
 - `main`: the extension's entry point run in `main` process.
 - `renderer`: the extension's entry point run in `renderer` process.
 - `engines.lens`: the minimum version of Lens API that the extension depends upon.
+  We only support the `~` range, which is also optional to specify, and only major and minor version numbers.
```

Comment:
lol, good to know, we probably need to update the generator template (which is using ^)

Reply:
Yeah, I mentioned the wrong range, oops.